tools/champ_blocs/cb_dissimilarity.xml

```diff
@@ -0,0 +1,113 @@
+<tool id="cb_dissim" name="Dissimilarity" version="@VERSION@" profile = "20.01">
+    <description>indicator</description>
+    <macros>
+        <import>macro.xml</import>
+    </macros>
+    <expand macro="cb_requirements">
+        <requirement type="package" version="2.16">r-rmarkdown</requirement>
+        <requirement type="package" version="0.2.4">r-officedown</requirement>
+        <requirement type="package" version="2.6_2">r-vegan</requirement>
+        <requirement type="package" version="1.3.2">r-tidyverse</requirement>
+        <requirement type="package" version="3.3.6">r-ggplot2</requirement>
+    </expand>
+    <required_files>
+        <include type="literal" path="test-data/champbloc_qecb.csv"/>
+        <include type="literal" path="test-data/ficheterrain.csv"/>
```

Comment:
please add the R files as well ... and if the image is needed, please the image as well.